vmaas/types.go

```diff
@@ -382,6 +388,13 @@ type ErratumDetail struct {
 	ReleaseVersions   []string `json:"release_versions"`
 }
 
+type PkgListItem struct {
+	Nevra       string     `json:"nevra"`
+	Summary     string     `json:"summary"`
+	Description string     `json:"description"`
+	Modified    *time.Time `json:"modified"`
```

Comment:
```suggestion
	Modified    *time.Time `json:"modified,omitempty"`
```
python API does not return `modified` attribute when `return_modified=false`